src/components/sizePicker/sizePicker.tsx

```diff
@@ -0,0 +1,64 @@
+import { forwardRef, useState } from 'react';
+
+import { Text } from '../text';
+import classes from './sizePicker.module.scss';
+
+interface SizePickerProps {
+  defaultValue: string;
+}
+
+export const SizePicker = forwardRef<HTMLDivElement, SizePickerProps>(
+  function SizePicker({ defaultValue }, ref) {
+    const [checked, setChecked] = useState(defaultValue);
+
+    return (
+      <div ref={ref}>
+        <Text
+          variant="poppinsRegular"
+          color="lightGray"
+          className={classes.title}
+        >
+          Size
+        </Text>
+        <div className={classes.optionsCont}>
+          <label>
+            <input
+              type="radio"
+              name="size"
+              value="S"
+              onChange={(e) => setChecked(e.target.value)}
+              checked={checked === 'S'}
+            />
+            <div className={classes.optionBox}>
+              <span className={classes.optionSize}>S</span>
```

Comment:
What if card don't have this size nor another? It's better to pass available options from props

`{
  options: [{value: 'S', label: 'S', disabled: false}]
}`